daemon/inspect.go

```diff
@@ -119,17 +119,19 @@ func (daemon *Daemon) getInspectData(container *container.Container, size bool)
 	}
 
 	containerState := &types.ContainerState{
-		Status:     container.State.StateString(),
-		Running:    container.State.Running,
-		Paused:     container.State.Paused,
-		Restarting: container.State.Restarting,
-		OOMKilled:  container.State.OOMKilled,
-		Dead:       container.State.Dead,
-		Pid:        container.State.Pid,
-		ExitCode:   container.State.ExitCode,
-		Error:      container.State.Error,
-		StartedAt:  container.State.StartedAt.Format(time.RFC3339Nano),
-		FinishedAt: container.State.FinishedAt.Format(time.RFC3339Nano),
+		Status:         container.State.StateString(),
+		Running:        container.State.Running,
+		Paused:         container.State.Paused,
+		Checkpointed:   container.State.Checkpointed,
+		Restarting:     container.State.Restarting,
+		OOMKilled:      container.State.OOMKilled,
+		Dead:           container.State.Dead,
+		Pid:            container.State.Pid,
+		ExitCode:       container.State.ExitCode,
+		Error:          container.State.Error,
+		StartedAt:      container.State.StartedAt.Format(time.RFC3339Nano),
+		FinishedAt:     container.State.FinishedAt.Format(time.RFC3339Nano),
+		CheckpointedAt: container.State.CheckpointedAt.Format(time.RFC3339Nano),
```

Comment:
nit, this will be formatted as the zero time here, probably it's worth checking if time is zero, if it is, set it to empty string, otherwise Format